ptvsd/wrapper.py

```diff
@@ -1180,10 +1191,38 @@ def on_stackTrace(self, request, args):
                 'line': line, 'column': 1,
             })
 
+        user_frames = []
+        for frame in stackFrames:
+            if not self.is_debugger_internal_path(frame['source']['path']):
+                user_frames.append(frame)
+
+        totalFrames = len(user_frames)
         self.send_response(request,
-                           stackFrames=stackFrames,
+                           stackFrames=user_frames,
                            totalFrames=totalFrames)
 
+    def is_debugger_internal_path(self, file_path):
```

Comment:
Can we move this out of this class, right now it feels like this class is doing way more than it needs to.

Reply:
This way, it gets easier to unit test the changes too (without having to run any live tests or instantiate this class).